src/read.c

```diff
@@ -3259,20 +3259,20 @@ static avifResult avifParseSampleDescriptionBox(avifSampleTable * sampleTable,
             return AVIF_RESULT_OUT_OF_MEMORY;
         }
         memcpy(description->format, sampleEntryHeader.type, sizeof(description->format));
-        size_t remainingBytes = sampleEntryHeader.size;
-        if ((avifGetCodecType(description->format) != AVIF_CODEC_TYPE_UNKNOWN)) {
-            if (remainingBytes < VISUALSAMPLEENTRY_SIZE) {
+        size_t sampleEntryBytes = sampleEntryHeader.size;
```

Comment:
nit: we could also mark this as `const` while we are at it.

Reply:
Done